liferay-content-management/liferay-portal
# LPD-32617 Change information panel sort to size, resolution and aspect ratio attributes

# What is this trying to solve?

https://liferay.atlassian.net/browse/LPD-32617

Sort information panel attributes, show first size, then resolution and finally aspect ratio. The rest of attributes are left as they are.

# How am I fixing it?

Sort information panel attributes, show first size, then resolution and finally aspect ratio. The rest of attributes are left as they are.

# How can you verify that it works?

Add a file in documents and media, then go to content dashboard and see the information panel for the added file. You should see the information as I wrote above.

Run the included automated tests.
